tests/test-helpers.js

```diff
@@ -79,7 +79,7 @@ export async function runCodemod(cwd) {
   await execaNode({
     cwd,
     stdio: 'inherit',
-  })`${updateScriptPath} --skip-git --skip-v2-addon`;
+  })`${updateScriptPath} --skip-git --skip-v2-addon ${codemodOptions}`;
```

Comment:
Now that we are actually generating TS apps, we also need to run the codemod with `--ts` options.